### ISSUE TITLE
[Other] Strategy Request: Copying pending PRs

### ISSUE THREAD

**Issue by qcasey** [paperless-ngx/paperless-ng#3]
I do not like the idea of simply merging all [44 remaining PRs](https://github.com/jonaswinkler/paperless-ng/pulls). Ideally those 44 PRs come over to this repo to be reviewed for conflicts, code quality, status checks, etc. before being merged/rejected.

I've started the oldest one (originating over two years ago!) at #2 

### My questions to anyone with an opinion:

1. #2 is how I plan to do the other 43. Manually requesting a new PR from the contributor's branch into paperless-ngx:dev. Does that sound right?
2. I did a little research and couldn't find an automated way of doing this. Is there something I'm missing? I don't mind doing it manually but would appreciate the hand from a robot too.

If I could get a couple opinions on this before starting, that would be great. After creating the PRs I'll tag in the relevant teams for review and testing.

**Comment by yschroeder** [comment]
I don't know if this can be automated. I would suggest to @ mention the author in the original PR and ask for a new PR in this repository. If they don't respond, we can go ahead and create them ourselves like #2. However, if we decide to not merge it would be unnecessary work for the author.

Maybe we should triage the PRs before asking to migrate?

**Comment by yschroeder** [comment]
Yes, bring them over like #2 and ping the original contributors if needed. We should give them a heads up in the original PR once we merged. This will also increase visibility of the new project and lead people over here.

**Comment by qcasey** [comment]
Love it

**Comment by qcasey** [comment]
> This sure can be automated

Do you have a link to that being done? I saw lots for merging all PRs from upstream, but not about copying the request itself

> Now that I think of it, it seems a good idea to move over all PRs and then either closing them, merging them or improving them.

Yeah that's what I was thinking :smiley:

**Comment by bauerj** [comment]
> Do you have a link to that being done? I saw lots for merging all PRs from upstream, but not about copying the request itself

I didn't find something ready for use either. If you'd like, I can write up a short script that does this tomorrow.

**Comment by qcasey** [comment]
Sure that would be cool! Thanks!
